### ISSUE TITLE
Unable to run dataimporter on Windows

### ISSUE THREAD

**Issue by aaronincincy** [kevinclement/SimpleArmory#417]
Windows 10 Pro, fresh install of Python 3, I get the following output when I run `python3 -m dataimporter --fixers pets`

```
INFO:root:Running fixer 'pets'.
Exception ignored in: <function _ProactorBasePipeTransport.__del__ at 0x00000260D7CA31C0>
Traceback (most recent call last):
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.752.0_x64__qbz5n2kfra8p0\lib\asyncio\proactor_events.py", line 116, in __del__
    self.close()
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.752.0_x64__qbz5n2kfra8p0\lib\asyncio\proactor_events.py", line 108, in close
    self._loop.call_soon(self._call_connection_lost, None)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.752.0_x64__qbz5n2kfra8p0\lib\asyncio\base_events.py", line 745, in call_soon
    self._check_closed()
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.752.0_x64__qbz5n2kfra8p0\lib\asyncio\base_events.py", line 510, in _check_closed
    raise RuntimeError('Event loop is closed')
RuntimeError: Event loop is closed
Exception ignored in: <function _ProactorBasePipeTransport.__del__ at 0x00000260D7CA31C0>
Traceback (most recent call last):
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.752.0_x64__qbz5n2kfra8p0\lib\asyncio\proactor_events.py", line 116, in __del__
    self.close()
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.752.0_x64__qbz5n2kfra8p0\lib\asyncio\proactor_events.py", line 108, in close
    self._loop.call_soon(self._call_connection_lost, None)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.752.0_x64__qbz5n2kfra8p0\lib\asyncio\base_events.py", line 745, in call_soon
    self._check_closed()
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.752.0_x64__qbz5n2kfra8p0\lib\asyncio\base_events.py", line 510, in _check_closed
    raise RuntimeError('Event loop is closed')
RuntimeError: Event loop is closed
```
From here the importer freezes and I have to break out of it.

**Comment by seirl** [comment]
I don't have access to a windows machine, but could you check if this patch fixes it?

```diff
diff --git dataimporter/__main__.py dataimporter/__main__.py
index 8419777..5022f58 100644
--- dataimporter/__main__.py
+++ dataimporter/__main__.py
@@ -3,6 +3,12 @@ import logging
 import json
 from pathlib import Path
 
+import platform
+import asyncio
+
+if platform.system() == 'Windows':
+    asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
+
 from .achievements import AchievementFixer
 from .mounts import MountFixer
 from .pets import PetFixer
```

**Comment by aaronincincy** [comment]
Sorry for the delayed response - that patch fixes my dataimporter issue and it runs successfully now.

**Comment by seirl** [comment]
Fixed in 1c75816
